group-redux/src/App.js

```diff
@@ -8,4 +18,21 @@ function App() {
   );
 }
 
+ReactDOM.render(
+  <React.StrictMode>
+    <Provider store={store}>
+      <BrowserRouter>
+        <Routes>
+          <Route>
+            <Route path="/" element={<App />} />
+            <Route path="dashboard" element={<Dashboard />} />
+            <App />
```

Comment:
you're invoking App and also calling app in a route.  Remove this line if you're going to do it in the route.